batch-index/build.gradle

```diff
@@ -2,8 +2,16 @@ dependencies {
     implementation project(':search-commons')
     implementation project(':indexing-handlers')
 
+    implementation group: 'com.github.bibsysdev', name: 'core', version: project.ext.nvaCommonsVersion
```

Comment:
Added all dependencies that before were declared in the commons as "api" dependencies